EasyLearn/app/views/player/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<%= stylesheet_link_tag "player", "data-turbo-track": "reload" %>
+
+  <nav>  
+    <h3 >Logo</h3>
+    <div class="menu">
+      <p>My events</p>
+      <div class="user-option">
+        <p class="username">username</p>
+        <%= image_tag("dropdown.png", :alt => "rss feed", :class => "profile-picture") %>
```

Comment:
I recommend use svg images, and create an `helper` for show it and render your images in your views